src/Notifier.cpp

```diff
@@ -11,10 +11,10 @@
 #include <Shotgun.h>
 #include <Vampire.h>
 #include <StopWatch.h>
+#include <czmq.h>
 
-std::unique_ptr<Notifier>  Notifier::CreateNotifier(const std::string& notifierQueue, const std::string& handshakeQueue, const size_t handshakeCount) {
-
-   auto notifier = std::unique_ptr<Notifier>(new Notifier(notifierQueue, handshakeQueue));
+std::unique_ptr<Notifier>  Notifier::CreateNotifier(const std::string& notifierQueue, const std::string& handshakeQueue, const size_t handshakeCount, const size_t maxTimeoutInSec, const int maxGetShotTimeoutInMs) {
+   auto notifier = std::unique_ptr<Notifier>(new Notifier(notifierQueue, handshakeQueue, maxTimeoutInSec, maxGetShotTimeoutInMs));
```

Comment:
It looks fine. One thing to keep in mind is that if you use initialer list instead of normal () then it will also catch type mismatches

    new Notifier { ....}

I.e no change needed, only recommendation for the future